_posts/2020/2021-01-07-how-to-animate-a-svg-with-css.md

```diff
@@ -0,0 +1,149 @@
+
+After finally exporting your illustration as an SVG, you can open it in a code editor and see the SVG code:
+
+{% image_custom image="/assets/img/pages/blog/images/illustration-svg.png" width="80" lightbox %}
+
+Now, you have the possibility to adjust the SVG directly in the code.
+You can customize classes and IDs according to your wishes or directly change the style rules.
+This is also a good way to edit SVGs that you didn't create yourself.
+
+Here’s a tip for optimizing your SVG after export:
+There are several tools that help you to keep your file as small and clean as possible, for example [SVGO](https://github.com/svg/svgo){:target="_blank"} or [SVG OMG](https://jakearchibald.github.io/svgomg/){:target="_blank"}.
+But be careful: When using such tools, you can over-optimize your SVG which could make it difficult to manipulate it with CSS later on.
+
+## How to animate your SVG with CSS
+
+Now, let's talk about how to bring your image to life. With CSS animations we can implement simple but also cool animations that serve many use cases.
+Let's try it out:
+
+Once you've added your SVG or SVG code to a page, you have it in the DOM, which allows you to target and style your SVG with CSS like any other HTML element.
+
+In our example, we just add a bit of CSS.
+As you can see, if we simply use the [animation property](https://css-tricks.com/almanac/properties/a/animation/){:target="_blank"} in CSS with a bit of [transform](https://css-tricks.com/almanac/properties/t/transform/){:target="_blank"} and opacity, we already have a cool and simple hover animation: [SVG CSS animation checkmark](https://codepen.io/zefia/pen/QWKpBop){:target="_blank"}.
+
+<iframe height="380" style="width: 100%;" scrolling="no" title="SVG CSS animation checkmark" src="https://codepen.io/zefia/embed/QWKpBop?height=380&theme-id=dark&default-tab=html,result" frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
+  See the Pen <a href='https://codepen.io/zefia/pen/QWKpBop'>SVG CSS animation checkmark</a> by Svenja Völz
+  (<a href='https://codepen.io/zefia'>@zefia</a>) on <a href='https://codepen.io'>CodePen</a>.
+</iframe>
+
+But as you can also see, there are some special characteristics in combination with the SVG.
+So, let's take a closer look at two things that I think are necessary to animate SVGs successfully.
+
+## How to transform an SVG
+
+Because of its own coordinate system, animating SVGs can be quite confusing, especially in the beginning.
+So, here is an overview of how to transform an SVG.
+
+HTML elements have their coordinate system originating at the 50% point of the reference box.
+In contrast, without specifying anything else, the origin of each SVG coordinate system is in the upper left corner (0,0) of the parent element.
+
+Let's have a look at an example.
+In this [SVG rotate origin](https://codepen.io/zefia/pen/abmyjMW){:target="_blank"}, we have not specified anything else.
```

Comment:
Do we still need the links to codepen if we embed it? I'd be fine with that but wanted to make sure. Somehow it's duplicated content